php/container.class.inc

```diff
@@ -291,23 +291,21 @@ class Container implements \JsonSerializable, \LORIS\Data\DataInstance
     /**
      * Sets the ID of the container's current centerId
      *
-     * @param int $centerId the ID of the container's current centerId
+     * @param \CenterID $centerId the ID of the container's current centerId
      *
      * @return void
      */
-    public function setCenterId(int $centerId) : void
+    public function setCenterId(\CenterID $centerId) : void
     {
-        if ($centerId > 0) {
-            $this->centerId = $centerId;
-        }
+        $this->centerId = $centerId;
     }
 
     /**
      * Gets the ID of the container's current centerId
      *
-     * @return ?int
+     * @return \CenterID
```

Comment:
@HenriRabalais just wanna check here that it makes sense to you to remove the null return type as I don't believe centerId can/should be null(?)